### ISSUE TITLE
Visualization specification: Is tsv the right choice?

### ISSUE THREAD

**Issue by dweindl** [PEtab-dev/PEtab#499]
Regarding the visualization specification in PEtab1.0:
I was wondering whether the tsv format was really a good choice there. Currently each line in the visualization table corresponds to one data series. The lines are grouped into plots based on `plotId`. For each data series one can set plot-level properties such as axis labels, axis scaling, etc. This makes it unnecessarily complex, as only one of those values can actually be used. Maybe this would be better done in YAML in a future version, which would be more able to account for a hierarchical specification of plotting properties? Any opinions?

**Comment by yannikschaelte** [comment]
I would agree that yaml may help avoid the redundancy/complexity. What is your opinion experts @plakrisenko @erikadudki @MerktSimon ?

**Comment by MerktSimon** [comment]
I would say, it is not really more complex to rewrite the plot level specifications for each line of a `plotId`. But it is definitely redundant.
Plus for tsv is imO that everyone can easily alter it with any office software without knowledge of syntax.
On the other hand yaml could allow for even more optional plot level specifications, maybe even general matplotlib kwargs, and hence more detailed reproducibility.
